lib/install/validate/storage.go

```diff
@@ -41,11 +40,6 @@ func (v *Validator) storage(ctx context.Context, input *data.Data, conf *config.
 		return
 	}
 
-	// provide a default path if only a DS name is provided
-	if imageDSpath.Path == "" {
```

Comment:
No wait, I'm wrong. We should be keeping the display name as the default. Backing this PR out.